src/support/utf8.c

```diff
@@ -615,71 +615,63 @@ size_t u8_printf(const char *fmt, ...)
     return cnt;
 }
 
-/* based on the valid_utf8 routine from the PCRE library by Philip Hazel
+/* Rewritten completely, original code not based on anything else
 
    length is in bytes, since without knowing whether the string is valid
    it's hard to know how many characters there are! */
-int u8_isvalid(const char *str, size_t length)
+int u8_isvalid(const char *iStr, size_t iLength)
```

Comment:
Yes, I'd seen that paragraph, but it says *nothing* about variable naming... and isn't followed in the code I've seen.